src/gpu-common.mk

```diff
@@ -1,6 +1,6 @@
 NVCC:=nvcc
 GPU_PTX_ARCH:=compute_35
-GPU_ARCHS?=sm_37,sm_50,sm_61,sm_70
+GPU_ARCHS?=sm_37,sm_50,sm_61,sm_70,sm_75,sm_80
```

Comment:
Remove sm_37 and sm_50? Each one increases the compilation and binary size.

Reply:
Done